terrain/instancing/voxel_instance_generator.cpp

```diff
@@ -507,7 +507,7 @@ void VoxelInstanceGenerator::generate_transforms(
 		bool surface_normal_is_normalized = false;
 		bool sphere_up_is_computed = false;
 		bool sphere_distance_is_computed = false;
-		float sphere_distance;
+		float sphere_distance = 0.f;
```

Comment:
`sphere_distance` is not always used, but cases where it is used have it initialized. Same story here: if it gets initialized here, won't that incur a cost when configurations such that it's not actually used? It's quite annoying